doc/man/cbmc.1

```diff
@@ -1,7 +1,4 @@
-.\" Process this file with
-.\" groff -man -Tascii cbmc.1
-.\"
-.TH CBMC 1 "JUNE 2014" "cbmc-4.7" "User Manual"
+.TH CBMC "1" "June 2022" "cbmc-5.59.0" "User Commands"
```

Comment:
Is there some way to get the version here to be integrated with the one in `src/config.inc`? Maybe some `sed` magic before a new version? 🤔